nwcsafpps_runner/tests/test_nwp_prepare.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2021 Adam.Dybbroe
+
+# Author(s):
```

Comment:
I think we should get rid of the Authors list in the headers and instead collect authors in an AUTHORS.md file like is done for Satpy and Pyspectral and other Pytroll projects. You could perhaps add an AUTHORS file and remove them here in the headers tha affect this PR at least?